changes.go

```diff
@@ -293,10 +298,14 @@ func (ch *UpgradeCharmChange) Args() (map[string]interface{}, error) {
 
 // Description implements Change.
 func (ch *UpgradeCharmChange) Description() []string {
-	series := ""
+	var series string
 	if ch.Params.Series != "" {
 		series = " for series " + ch.Params.Series
 	}
+	var channel string
+	if ch.Params.Channel != "" {
+		channel = " for channel " + ch.Params.Channel
```

Comment:
Does this match the output of `juju deploy --channel`? IIRC we specify the channel in parenthesis. Either way works but we should be consistent.

Reply:
We swap `in` and `from` depending on the context.

Reply:
But yeah it's consistent in form.